### DISCUSSION TITLE
How to run more commands one by one?

### DISCUSSION THREAD

**Discussion by danli349** [nushell/nushell#14563] [General]
Hello:

In bash I can run a couple of commands across more lines using `&& \` to combine the commands, how can I do this in nushell?

Thanks a lot

**Comment by fdncred** [comment]
https://www.nushell.sh/book/coming_from_bash.html

**Comment by lemorage** [comment]
Use semicolons. `command1; command2` will do the trick.

**Comment by Lockszmith-GH** [comment]
Take a look at a recent discussion on a similar topic #6667

What I recall from that discussion was, semicolon (`;`), by default acts similar to `&&`, because the default behavior is equivalent to `set -e` in bash - where an error breaks execution.
